src/scripts/main.js

```diff
@@ -1,3 +1,234 @@
 'use strict';
 
-// write code here
+// create and sort table
+const table = document.querySelector('table');
+const tBody = document.querySelector('tbody');
+const headCells = table.rows[0].children;
+
+// click on thead
+table.tHead.addEventListener('click', action => {
+  const index = [...headCells].indexOf(action.target);
+  const newOrder = [...tBody.children];
+  const targetCell = action.target.closest('th');
+  const headCellCondition = targetCell.classList.contains('js-active-head');
```

Comment:
AFAIK headCellCondition is the sorting order, if yes, please rename, so it become more obvious

Reply:
let's store a boolean value, instead of assigning and removing a class on every click